pkg/proxy/util/utils.go

```diff
@@ -56,3 +59,45 @@ func ShouldSkipService(svcName types.NamespacedName, service *api.Service) bool
 	}
 	return false
 }
+
+type NetlinkHandle interface {
+	AddrAdd(link netlink.Link, addr *netlink.Addr) error
+	AddrDel(link netlink.Link, addr *netlink.Addr) error
+	LinkByName(dev string) (netlink.Link, error)
+}
+
+// EnsureAddressBind checks if address is bound to the interface and, if not, binds it. If the address is already bound, return true.
+func EnsureAddressBind(address, devName string, h NetlinkHandle) (exist bool, err error) {
+	dev, err := h.LinkByName(devName)
+	if err != nil {
+		return false, fmt.Errorf("error get interface: %s, err: %v", devName, err)
+	}
+	addr, err := netlink.ParseAddr(address + "/32")
```

Comment:
I think we can remove the /32 now. In fact, IPVS interface support IPV6 as well.